dependencies.xml

```diff
@@ -16,6 +16,8 @@ Note all install methods after "main" take
  add library from forge:     <library source='forge'>...
  optional:                   <library optional='True'>...
  skip run/install check:     <library skip_check='True'>...
+ add pip library with extra parameters:
+                             <ray source="pip" pip_extra="[tune]">1.9</ray>
```

Comment:
Could you add a little bit more descriptions about this feature? Maybe add an example
```
pip install -U "ray[tune]"  # installs Ray + dependencies for Ray Tune
pip install -U "ray[rllib]"  # installs Ray + dependencies for Ray RLlib
pip install -U "ray[serve]"  # installs Ray + dependencies for Ray Serve
```

Reply:
Good point. I expanded the example.